components/bl-social-login-button-component/component.json

```diff
@@ -0,0 +1,48 @@
+{
+  "id": "c_77c399a94abeab643336805b91a3aefb",
+  "name": "Social Login Button",
+  "description": "The Social Login Button component is designed for quick user login.",
+  "showInToolbox": true,
+  "faIcon": "toggle-on",
+  "mainJS": "dist/index.js",
+  "type": "custom",
+  "category": "Custom Components",
+  "properties": [
+    {
+      "type": "select",
+      "name": "buttonOption",
+      "label": "Button option",
+      "showInSettings": true,
+      "defaultValue": "googleplus",
+      "hasLogicHandler": false,
+      "handlerId": "buttonOptionLogic",
+      "options": [
```

Comment:
I thought this would be a good solution as the user can choose both the order of the buttons and the number of them.
If the ability to choose the number of buttons should be mandatory, then i can try to do it by checkboxes, what do you think?

Reply:
Yes, checkboxes will be good. In a case when user want's to change order, he may duplicate component. But the basic behavior must be - one component = multiple logins.